src/Biomass/FarmValuesTable.js

```diff
@@ -132,174 +69,342 @@ const FarmValuesTable = ({
-                        setCurrentIndex(index);
-                      }}
-                    >
-                      {open ? <Comment /> : <Comment />}
-                    </IconButton>
-                  </TableCell>
-                  <TableCell align="left">{record.code}</TableCell>
-                  <TableCell align="right">{record.subplot}</TableCell>
-                  <TableCell align="center">
-                    {record.cc_termination_date
-                      ? format(parseISO(record.cc_termination_date), 'PP')
-                      : 'N/A'}
-                  </TableCell>
-                  <TableCell align="center">{record.cc_species || 'No Data'}</TableCell>
-                  <TableCell
-                    align="center"
-                    title={`Uncorrected cover crop dry biomass in ${units}`}
-                  >
-                    {units === 'kg/ha'
-                      ? record.uncorrected_cc_dry_biomass_kg_ha
-                      : record.uncorrected_cc_dry_biomass_lb_ac}
-                  </TableCell>
-                  <TableCell
-                    align="center"
-                    title={`Ash corrected cover crop dry biomass in ${units}`}
-                  >
-                    {units === 'kg/ha'
-                      ? record.ash_corrected_cc_dry_biomass_kg_ha
-                      : record.ash_corrected_cc_dry_biomass_lb_ac}
-                  </TableCell>
-                  <TableCell align="center">{record.percent_n_nir}</TableCell>
-                  <TableCell align="center">{record.c_n_ratio}</TableCell>
-                  <TableCell align="center">{record.percent_carbohydrates}</TableCell>
-                  <TableCell align="center">{record.percent_cellulose}</TableCell>
-                  <TableCell align="center">{record.percent_lignin}</TableCell>
-                </CustomTableRow>
+  const tableHeaderOptions = [
+    {
+      title: 'Code',
+      field: 'code',
+      type: 'string',
+      align: 'justify',
+    },
+    {
+      field: 'subplot',
+      title: 'Rep',
+      type: 'numeric',
+    },
+    {
+      field: 'cc_termination_date',
+      title: 'Termination Date',
+      type: 'date',
+      render: (rowData) => {
+        return rowData.cc_termination_date
+          ? new Date(rowData.cc_termination_date).toLocaleDateString()
+          : 'N/A';
+      },
+    },
+    {
+      field: 'cc_species',
+      title: 'Cover Crop Species',
+      type: 'string',
+      render: (rowData) => {
+        return rowData.cc_species ? truncateCoverCrop(rowData.cc_species) : 'No Data';
+      },
+    },
+    {
+      field: 'uncorrected_cc_dry_biomass_kg_ha',
+      title: 'Uncorrected Dry Weight',
+      type: 'numeric',
+      render: (rowData) => {
+        return rowData.uncorrected_cc_dry_biomass_kg_ha
+          ? units === 'kg/ha'
+            ? Math.round(rowData.uncorrected_cc_dry_biomass_kg_ha)
+            : Math.round(rowData.uncorrected_cc_dry_biomass_kg_ha * 0.892179)
```

Comment:
do we want these coefficients extended @brianwdavis?

Reply:
Extended to what? I think rounding to the nearest integer on something in the 100-10,000 order is good.

Reply:
4 digits is plenty for this purpose

Reply:
yes i can